server/handlers/handlers.go

```diff
@@ -333,10 +333,18 @@ func (ah *ActivityHandler) handleUpdatedActivity(activityIds msteams.ActivityIds
 			return
 		}
 	} else {
-		post, err := ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
-		if err != nil {
-			ah.plugin.GetAPI().LogError("Unable to find the original post", "error", err.Error())
-			return
+		post, postErr := ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
+		if postErr != nil {
+			if strings.Contains(postErr.Error(), "Unable to get the post.") {
```

Comment:
The error is better compared by the Error.Id, instead of the `Error` string, that can be translated.